components/dialog/dialog-mixin.js

```diff
@@ -1,4 +1,5 @@
 import '../focus-trap/focus-trap.js';
+import '../../helpers/viewport-size.js';
```

Comment:
I noticed in fullscreen dialogs that `--d2l-vh` wasn't defined vs. in regular dialogs it was, which was what was causing #2200. All we need to do is include this in both places (i.e. move it to the mixin) and bingo!